Core/clim-core/frames.lisp

```diff
@@ -496,9 +496,13 @@ documentation produced by presentations.")
               (*partial-command-parser* partial-command-parser)
               (interactorp (typep *query-io* 'interactor-pane)))
          (restart-case
-             (progn
-               (redisplay-frame-panes frame :force-p first-time)
-               (setq first-time nil)
+             (labels ((exec-cmd (frame command)
```

Comment:
Why does this function accept a `FRAME` argument? It's already available in the outer scope.